source/installguide/configuration.rst

```diff
@@ -512,7 +512,7 @@ Basic Zone Configuration
    -  **Name.** The name of the storage device.
 
    -  **Protocol.** For XenServer, choose either NFS, iSCSI, or
-      PreSetup. For KVM, choose NFS, SharedMountPoint,CLVM, RBD or custom (for PowerFlex). For
+      PreSetup. For KVM, choose NFS, SharedMountPoint,CLVM, RBD, FiberChannel or custom (for PowerFlex). For
```

Comment:
The naming is not ideal, as fibrechannel can also (has been historically) configured outside of ACS, manuially, and just a Shared Mount Point would be feed to ACS. But leave it as it is.